cfme/infrastructure/datastore.py

```diff
@@ -210,7 +209,7 @@ def get_hosts(self):
         view = navigate_to(self, 'Details')
         view.contents.relationships.click_at('Hosts')
         hosts_view = view.browser.create_view(RegisteredHostsView)
-        return [h.name for h in hosts_view.entities.get_all()]
+        return hosts_view.entities.get_all()
```

Comment:
This is changing from returning list of host names to list of host entitiy objects - I think you want to call `all_entity_names()` here.

Reply:
:+1: yes, good idea. this PR was finished before you added that method.

Reply:
Though it looks like you've changed callers to handle getting entity objects back, not just names. Forget previous comment, because I think its _better_ to return entity objects here, not just the names.